integration-tests/api/__tests__/admin/product-category.ts

```diff
@@ -462,4 +467,106 @@ describe("/admin/product-categories", () => {
       )
     })
   })
+
+  describe("POST /admin/product-categories/:id/products/batch", () => {
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+
+      productCategory = await simpleProductCategoryFactory(dbConnection, {
+        id: "test-category",
+        name: "test category",
+      })
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("should add products to a product category", async () => {
+      const api = useApi()
+      const testProduct1 = await simpleProductFactory(dbConnection, {
+        id: "test-product-1",
+        title: "test product 1",
+      })
+
+      const testProduct2 = await simpleProductFactory(dbConnection, {
+        id: "test-product-2",
+        title: "test product 2",
+      })
+
+      const payload = {
+        product_ids: [{ id: testProduct1.id }, { id: testProduct2.id }],
+      }
+
+      const response = await api.post(
+        `/admin/product-categories/${productCategory.id}/products/batch`,
+        payload,
+        adminHeaders
+      )
+
+      expect(response.status).toEqual(200)
+      expect(response.data.product_category).toEqual(
+        expect.objectContaining({
+          id: productCategory.id,
+          created_at: expect.any(String),
+          updated_at: expect.any(String),
+        })
+      )
+
+      const products = await dbConnection.manager.find(Product, {
```

Comment:
todo: I think that we should have a route /product-categories/:id/products to be able to retrieve the product from specific categories and that we can paginate. wdyt?

Reply:
thought: We can also have the product list end point that accepts a collection of category_ids to retrieve the product filtered by multiple categories.

cc: @olivermrbl for both above comments

Reply:
@adrien2p there is an existing issue for this, which will come up a couple of PRs later.